db/seeds.rb

```diff
@@ -10,6 +10,14 @@
 
 User.skip_callback(:validation, :before, :generate_password)
 
+Country.create([{ name: 'Belarus' },
+                            { name: 'Albania' },
+                            { name: 'Algeria' },
+                            { name: 'Poland' },])
+Country.last.cities.create([{ name: 'Warsaw ' },
+                            { name: 'Krakow' },
+                            { name: 'Zakopane' },
+                            { name: 'Gdańsk' },])
```

Comment:
```suggestion
Country.find_by(name: 'Poland').cities.create([{ name: 'Warsaw ' },
                            { name: 'Krakow' },
                            { name: 'Zakopane' },
                            { name: 'Gdańsk' }])
```
использовать last не самый надежный вариант